tests/snmp/test_snmp_v2mib.py

```diff
@@ -0,0 +1,46 @@
+"""
+Test SNMPv2MIB in SONiC.
+"""
+
+import pytest
+from tests.common.helpers.assertions import pytest_assert # pylint: disable=import-error
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+
+def test_snmp_v2mib(duthosts, rand_one_dut_hostname, localhost, creds):
```

Comment:
we have snmp tests, do you know which tests are not covered by existing tests?

Reply:
@lguohan I didn't find test that would cover snmp sysDescr MIB, also I added verification of sysName, sysLocation and sysContact MIBs.